testgrid/config/yaml2proto/yaml2proto_test.go

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yaml2proto
+
+import (
+	"testing"
+	"io/ioutil"
+
+	"github.com/golang/protobuf/proto"
+	pb "k8s.io/test-infra/testgrid/config/pb"
+)
+
+func TestYaml2ProtoEmpty(t *testing.T) {
+	yamlData, err := ioutil.ReadFile("testdata/Empty.yaml")
```

Comment:
Is there a good reason for Empty.yaml to live in another file rather than just defining it inline here?